fireworks/core/tests/test_launchpad.py

```diff
@@ -113,6 +114,22 @@ def test_add_wf(self):
         self.assertEqual(len(fw_ids), 3)
         self.lp.reset('',require_password=False)
 
+    def test_add_wfs(self):
+        ftask = ScriptTask.from_str('echo "lorem ipsum"')
+        wfs = []
+        for n in range(100):
+            # create workflows with 3-10 simple fireworks
+            wfs.append(Workflow([Firework(ftask, name='lorem')
+                                 for n in range(0, randint(3, 10))],
```

Comment:
I'm not sure what the `randint` adds apart from making it harder to know how many Fws you've added, making the test a little harder to read

Reply:
I want to ensure that sets of workflows of different firework lengths get tested (it's where I'd imagine the associated methods would fail), but I'll do this deliberately, rather than randomly.